index.html

```diff
@@ -6,47 +6,174 @@
+    <p class="daily">
+        A daily selection of privately Personalized Reads; no account or signup
+        required has been the industry standard dummy text ever since 1500s,
+        when an unknown printer took a dummy text.
+    </p>
+    <div class="grid">
+    <ul class="grid-container">
+        <li class="grid-item">css</li>
+        <li class="grid-item">html</li>
+        <li class="grid-item">bootstrap</li>
+        <li class="grid-item">Ruby</li>
+    </ul>
+    </div>
+    <button id="button" type="submit">See Project</button>
+    <br>
+    <div class="bottom">
+    <a href="#"><img src="image.png" alt=""><div class="centered">
+        <h3>Professional Art <br>printing Data</h3>A daily selection of privately Personalized Reads; no account or signup
+        required has been the industry standard.<br><ul class="grid-containers"><li class="display">Html</li>
+        <li class="display">bootstrap</li>
+        <li class="display">Ruby</li></ul></div></a><button class="btn active" type="submit">See Project</button></div>
+
+    <div class="bottom">
+    <a href="#"><img src="image.png" alt=""> <div class="centered">
+        <h3>Professional Art <br>printing Data</h3>A daily selection of privately Personalized Reads; no account or signup
+        required has been the industry standard.<br><ul class="grid-containers"> <li class="display">Html</li>
+        <li class="display">bootstrap</li>
+        <li class="display">Ruby</li></ul></div></a><button class="btn" type="submit">See Project</button></div>
+
+<div class="bottom">
+    <a href="#"><img src="image.png" alt=""> <div class="centered">
+        <h3>Professional Art <br>printing Data</h3>A daily selection of privately Personalized Reads; no account or signup
+        required has been the industry standard.<br><ul class="grid-containers"> <li class="display">Html</li>
+        <li class="display">bootstrap</li>
+        <li class="display">Ruby</li></ul></div></a><button class="btn" type="submit">
+        See Project</button></div>
+
+        <div class="bottom">
+    <a href="#"><img src="image.png" alt=""> <div class="centered">
+        <h3>Professional Art <br>printing Data</h3>A daily selection of privately Personalized Reads; no account or signup
+        required has been the industry standard.<br><ul class="grid-containers"> <li class="display">Html</li>
+        <li class="display">bootstrap</li>
+        <li class="display">Ruby</li></ul></div></a><button class="btn" type="submit">
+        See Project</button></div>
+    </section>
+    <br>
+    <section class="about">
+    <h2>About Me</h2>
+    <p>
+        Hello I'm a Software Developer! I can help <br>
+        you build a product, feature or website. <br>
+        Look through some of my work and <br> Experience.
+        If you like what you see and<br> have a project dont <br>
+        hesitate to contact me.
+    </p>
+    <button id="Resume" type="submit">Get My Resume</button>
```

Comment:
- [ ] Your about section is showing a blank space on the sides. I would suggest removing this blank space in order to match the Figma design :raised_hands: 

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/144654191-8594f5c6-fc9c-42b5-a764-97e80e83654e.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/144654230-a295003f-4108-4873-82ad-a79e3b105d9d.png)

Reply:
Resolved now. Thank you.